string name #name of joint
uint8 id  #id on bus
string model_name #model name
uint8 error_code # error code
float64 voltage #current voltage
float64 temperature #current temperature